UNRELEASED.md

```diff
@@ -1,3 +1,13 @@
 # Unreleased
 
-__Nothing yet!__
+**Added:**
+
+- bpk-svgs
+  - New icons added:
+  - bws--call-back
+  - bws--end-call
+  - bws--keypad
+  - bws--mute
+  - bws--phone-call
+  - bws--send-message
+  - bws--un-mute
```

Comment:
This affects quite a lot of packages so we usually list those too in the changelog - see here for an example: https://github.com/Skyscanner/backpack/blob/master/CHANGELOG.md#2018-07-12---new-icons-and-better-close-link

Reply:
done 👍